apps/docs/src/stories/Icons.stories.tsx

```diff
@@ -7,667 +40,30 @@ export const Default = {
-          <div>
-            <Icons.IconClock style={style} />
-            <p>clock</p>
-          </div>
-        </div>
-
-        <div className='icons-group'>
-          <h4>Users</h4>
-          <div>
-            <Icons.IconUserCheck style={style} />
-            <p>user-check</p>
-          </div>
-          <div>
-            <Icons.IconUserEdit style={style} />
-            <p>user-edit</p>
-          </div>
-          <div>
-            <Icons.IconUserMinus style={style} />
-            <p>user-minus</p>
-          </div>
-          <div>
-            <Icons.IconUserPlus style={style} />
-            <p>user-plus</p>
-          </div>
-          <div>
-            <Icons.IconUserX style={style} />
-            <p>user-x</p>
-          </div>
-          <div>
-            <Icons.IconUser style={style} />
-            <p>user</p>
-          </div>
-          <div>
-            <Icons.IconUsersCheck style={style} />
-            <p>users-check</p>
-          </div>
-          <div>
-            <Icons.IconUsersMinus style={style} />
-            <p>users-minus</p>
-          </div>
-          <div>
-            <Icons.IconUsersPlus style={style} />
-            <p>users-plus</p>
-          </div>
-          <div>
-            <Icons.IconUsersX style={style} />
-            <p>users-x</p>
-          </div>
-          <div>
-            <Icons.IconUsers style={style} />
-            <p>users</p>
-          </div>
-        </div>
-
-        <div className='icons-group'>
-          <h4>Logo</h4>
-          <div>
-            <Icons.IconApple style={style} />
-            <p>apple</p>
-          </div>
-          <div>
-            <Icons.IconBehance style={style} />
-            <p>behance</p>
-          </div>
-          <div>
-            <Icons.IconFacebook style={style} />
-            <p>facebook</p>
-          </div>
-          <div>
-            <Icons.IconGithub style={style} />
-            <p>github</p>
-          </div>
-          <div>
-            <Icons.IconGoogleColor style={style} />
-            <p>google-color</p>
-          </div>
-          <div>
-            <Icons.IconGoogleMono style={style} />
-            <p>google-mono</p>
-          </div>
-          <div>
-            <Icons.IconInstagram style={style} />
-            <p>instagram</p>
-          </div>
-          <div>
-            <Icons.IconKakao style={style} />
-            <p>kakao</p>
-          </div>
-          <div>
-            <Icons.IconLinkedin style={style} />
-            <p>linkedin</p>
-          </div>
-          <div>
-            <Icons.IconPlaystore style={style} />
-            <p>playstore</p>
-          </div>
-        </div>
+        {iconCategories.map((category) => (
+          <div key={category.name} className='icons-group'>
+            <h4>{category.name}</h4>
+            {Object.values(category.icons).map((icon, index) => (
+              <IconGenerator icon={icon} style={style} key={`${category.name}-${index}`} />
+            ))}
```

Comment:
여기를 `Object.values`가 아니라 `Object.entries`를 쓰면 여기서 컴포넌트 이름도 받을 수 있을 것 같아요.
그 컴포넌트 이름을 `IconGenerator`에 넘겨주면 아래 로직도 안써도 되고 
```ts
const iconName = convertIconNameToKebabCase(icon.__docgenInfo.displayName);
```
`any` type도 안써도 될 것 같아요.

```suggestion
 {Object.entries(category.icons).map(([iconName, IconComponent]) => (
        <IconGenerator key={iconName} icon={IconComponent} name={iconName} style={style} />
      ))}
```

또한 `__docgenInfo` 자체도 코드로 직접 작성하는 것보다 storybook이 내부적으로 docs를 위해서 사용하는 값 같아서 이게 실제 배포환경이나 빌드 됐을때 문제가 없을지 조금 걱정이 되기도 합니다.
_(이 부분은 저도 지식이 부족해 아는 게 있으시다면 알려주세요!)_